### PR TITLE
Bump current flow spec version to 13.3

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/utils/management/commands/mailroom_db.py [hunk 1]
```diff
@@ -399,13 +399,14 @@ def create_templates(self, spec, org, templates):
...
-                    content=tt["content"],
-                    variable_count=tt["variable_count"],
                     status=tt["status"],
                     external_id=tt["external_id"],
                     external_locale=tt["external_locale"],
                     namespace=tt["namespace"],
                     components=tt["components"],
+                    # deprecated
+                    content=tt["content"],
+                    variable_count=tt["variable_count"],
```
Comment (rowanseymour) [inline temba/utils/management/commands/mailroom_db.py:409]: unrelated

**Review by norkans7 — APPROVED** [review]
